### DISCUSSION TITLE
An overview of the number of components and stories documented in Storybook

### DISCUSSION THREAD

**Discussion by nielsvanmidden** [storybookjs/storybook#29750] [Ideas]
### Describe the solution you'd like

I'd like to have an overview of the number of components and stories documented in Storybook for several reasons:

**Progress Tracking**
It helps monitor how much of your design system is covered, identifying gaps or under-documented areas.

**Team Communication**
Provides a clear snapshot for stakeholders or team members about the current documentation/adoption status.

**Scalability**
As your design system grows, understanding its size can help plan resources for maintenance and updates.

**Quality Assurance**
Ensures all components are properly documented, reducing risks of inconsistent usage or development errors.

### Are you able to assist to bring the feature to reality?

no

**Comment by shilman** [comment]
Good one! 💯

**Comment by domyen** [comment]
We could start with something straightforward like adding an "Overview" doc in the Storybook sidebar. 

Clicking on "Overview" would take you to a page which prints:
- Component count
- Story count
- Type of test dimension and status for each test

I think progress tracking and team communication would be tougher for Storybook to report on because those are more of an organizational (aka human) thing.

<img width="507" alt="image" src="https://github.com/user-attachments/assets/429b9e26-79b0-443f-964b-e4b6d0e1ca25">
<img width="882" alt="image" src="https://github.com/user-attachments/assets/6da65083-12d8-4448-9ea1-483500c44ef6">

**Reply by nielsvanmidden** [reply]
Lovely!
